### ISSUE TITLE
Change the client geocoder reference in configuration

### ISSUE THREAD

**Issue by ethervoid** [CartoDB/dataservices-api#108]
The `geocoder_config_server` should be changed and use `dataservices_config_server` instead.

This change implies remove the old configuration from all the users and put the new one as well as change the editor part too.

**Comment by rafatower** [comment]
Is this done? I cannot find any reference to those names in any repo...

**Comment by ethervoid** [comment]
Nope, is not done. Here is the guilty part https://github.com/CartoDB/cartodb/blob/master/app/models/user/db_service.rb#L1421-1422

It's a change that should be make in two phases, first add the new config key and change the client part to use it. And then remove the old one.

**Comment by rafatower** [comment]
ok, just updated the description.
